test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -23,13 +23,51 @@
 using namespace std;
 using namespace yul::test::yul_fuzzer;
 
+std::string const yul::test::yul_fuzzer::createHex(std::string const& _hexBytes)
+{
+	std::string tmp{_hexBytes};
+	if (!tmp.empty())
+	{
+		tmp.erase(std::remove_if(tmp.begin(), tmp.end(), [](char c) {
+			return !std::isxdigit(c);
+		}), tmp.end());
+		tmp = tmp.substr(0, 64);
+	}
+	// We need this awkward if case because tmp can also be null post erase call above
+	if (tmp.empty())
+		tmp = "1";
+	return tmp;
+}
+
+std::string const yul::test::yul_fuzzer::createAlphaNum(std::string const& _strBytes)
```

Comment:
Please remove the `const` so it will be moved properly.

Reply:
ack